### PR TITLE
docs(storybook): guide for setting up storybook composition

### PR DESCRIPTION
**Pull request by mandarini**
## Expected Behavior
Short guide of how to set up Storybook composition on an Nx workspace.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/shared/guides/storybook/storybook-composition-setup.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+You can always read more in the [official Storybook docs](https://storybook.js.org/docs/angular/workflows/storybook-composition#compose-published-storybooks).
+
+## How to use it in Nx
+
+It's quite easy to use this feature, in Nx and in general, since you do not need to make any code changes, you just need to have the "composed" Storybook instances (the ones you need to "compose") running, choose a "host" Storybook, and just add the composed Storybooks in it's `.storybook/main.js` file.
+
+Nx provides the [`run-many`](https://nx.dev/l/a/cli/run-many) command, which will allow you to easily run multiple Storybooks at the same time. You need to run the `run-many`
+command with the parallel flag set to true `--parallel=true`, because you want to run all your Storybooks in parallel. Since the default amount of parallel activities that you can run
```
Comment (vsavkin) [inline docs/shared/guides/storybook/storybook-composition-setup.md:48]: Can you change it to `--parallel=3`? We are deprecating --parallel=BOOLEAN and only keeping --parallel=NUMBER

File: docs/map.json [hunk 2]
```diff
@@ -871,6 +871,11 @@
             "name": "Upgrading to Storybook 6 (Angular)",
             "file": "shared/guides/storybook/storybook-v6-angular"
           },
+          {
+            "id": "storybook-composition-setup",
+            "name": "USetting up Storybook Composition with Nx",
```
Comment (juristr) [inline docs/map.json:876]: there's a typo in the `name` => "USetting...`

**Review by juristr — CHANGES_REQUESTED** [review]

File: docs/shared/guides/storybook/storybook-composition-setup.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+In essence, you have a Storybook running, which will be the host of the embeded Storybooks as well. Then, you provide this "host" Storybook with a URL of a live/running Storybook. The composed Storybook is then displayed in a new Canvas iframe as part of the host Storybook, and is listed on the left-hand-side stories inventory, too. You can read more about this in the docs listed above.
+
+## How to use it
+
+All you need is a URL of a live Storybook, and a "host" Storybook. In the `.storybook/main.js` file of the "host" Storybook, inside `module.exports` you add a new `refs` attribute, which will contain the link(s) for the composed Storybook(s).
+
+In the example below, we have a host Storybook running on local port 4400 (http://localhost:4400) - not displayed here. In it, we want to compose three other Storybooks. The "one-composed" and "two-composed", running on local ports `4401` and `4402` accordingly, as well as the [Storybook website's Storybook](https://next--storybookjs.netlify.app/official-storybook) which is live on the address that you see.
+
+```
```
Comment (juristr) [inline docs/shared/guides/storybook/storybook-composition-setup.md:20]: Can you add the language flavor to the code embeds? That way it gets syntax highlighted :)

**Comment by mandarini** [comment]
Updated PR with changes

**Review by juristr — APPROVED** [review]
LGTM

**Pull request merged**